docs/en/observability/profiling-upgrade.asciidoc

```diff
@@ -5,6 +5,9 @@
 <titleabbrev>Upgrade</titleabbrev>
 ++++
 
+This section is specific to upgrading Universal Profiling when running it in {ecloud} or ECE.
+For self-hosted installations, refer to <<profiling-upgrade-self-hosted,Upgrade Universal Profiling in self-hosted installations>>.

```

Comment:
```suggestion
For self-hosted installations, refer to <<profiling-self-managed-upgrade,Upgrade Universal Profiling in self-hosted installations>>.

```
Looks like this is why it's failing the build. I'll take a closer look at the rest of the doc here shortly.